packages/fast-development-site-react/README.md

```diff
@@ -1,2 +1,86 @@
 # FAST React development site
-This is the development site which can be used to create a react environment for a group of react or web components.
\ No newline at end of file
+This is the development site that can be used to create a React site for a library of React components.
```

Comment:
I'd like us to explain more in the context of why the user/contributor would want to use this, what do they get?

Reply:
Resolved